src/clr-angular/utils/drag-and-drop/index.ts

```diff
@@ -8,11 +8,11 @@ import {Type} from "@angular/core";
 import {ClrDragHandle} from "./drag-handle";
 import {ClrDraggableGhost} from "./draggable-ghost";
 import {ClrDraggable} from "./draggable/draggable";
-import {ClrDroppable} from "./droppable";
+import {ClrDroppable} from "./droppable/droppable";
 import {ClrIfDragged} from "./if-dragged";
 
 export * from "./draggable/draggable";
-export * from "./droppable";
+export * from "./droppable/droppable";
 export * from "./if-dragged";
 export * from "./drag-handle";
 export * from "./draggable-ghost";
```

Comment:
I just realized 2 things:
- We need an Angular module for drag and drop, which you're missing.
- You need to export the public event type (not the private one).

Reply:
>We need an Angular module for drag and drop, which you're missing

```
@NgModule({
    imports: [CommonModule],
    declarations: [DRAG_AND_DROP_DIRECTIVES],
    entryComponents: [ClrDraggableGhost],
    providers: [ClrDragAndDropEventBus],
    exports: [DRAG_AND_DROP_DIRECTIVES]
})
export class ClrDragAndDropModule {}
```
We do have a modul for drag and drop. Or are you talking about something else?

Reply:
Oops, I don't know what I was thinking. I forgot it was a partial contribution on an already well-advanced topic branch, sorry about that. My comment on the public event still stands though.

Reply:
No worries :D I added the public event to the `index.ts` file.